### ISSUE TITLE
Option to select specific OpenCL and CUDA Device Indices

### ISSUE THREAD

**Issue by cxhernandez** [rmcgibbo/openmm-bench#3]
Especially useful for machines with multiple GPUs. Right now I'm just manually editing the `./benchmark/*.py` files for each machine.

**Comment by rmcgibbo** [comment]
We kind of want the single machine to show up as two machines, right? I'm not exactly sure how to accomplish this with asv.

**Comment by cxhernandez** [comment]
Oh, this is something that I imagine would be included in `~/.asv-machine.json` and passed as an argument at runtime.

**Comment by rmcgibbo** [comment]
Yeah, that seems reasonable, but it's not currently how things are set up.
